tests/integration/annotation_import/test_model_run.py

```diff
@@ -117,7 +117,7 @@ def test_model_run_export_labels(model_run_with_model_run_data_rows):
     assert len(labels) == 3
 
 
-@pytest.mark.skip(reason="feature under development")
+# @pytest.mark.skip(reason="feature under development")
```

Comment:
You can just delete this line instead of commenting it out

Reply:
Although I think exports v2 is broken in staging right now.. Maybe leave this until it is more stable.